app/javascript/components/patient/Patient.js

```diff
@@ -23,6 +23,7 @@ class Patient extends React.Component {
       expandPlannedTravelNotes: false,
       primaryLanguageDisplayName: null,
       showSetFlagModal: false,
+      isMinor: props.details && props.details.date_of_birth && isMinor(props.details.date_of_birth),
```

Comment:
one last comment... I don't think you need this in the state anymore because of the util method, you could just do this inline (since it's only twice)